pkg/services/live/events.go

```diff
@@ -0,0 +1,18 @@
+package live
+
+// DashboardEventActionSaved action when dashbord is saved
+var DashboardEventActionSaved = "saved"
+
+// DashboardEventActionEditing action when dashbord is edited (sent from client)
+var DashboardEventActionEditing = "editing"
+
+// DashboardEventActionDeleted action when dashbord is deleted
+var DashboardEventActionDeleted = "deleted"
```

Comment:
for enums we do something like this usually https://github.com/grafana/grafana/blob/master/pkg/models/org_user.go#L18